Hec.Cwms/CwmsDatabase.cs

```diff
@@ -47,14 +49,13 @@ public enum StorageRule {
     /// --exec cwms_env.set_session_office_id('NAB');
     /// </summary>
     /// <param name="officeId"></param>
```

Comment:
The comments indicates a argument.

Reply:
How should I address this?

Reply:
To use the CWMS database you usually need to know an officeID.   This could be communicated by a CwmsDatabase constructor that requires an officeID.

Reply:
One possibility is to have  SetOfficeID(string id) set the officeID in the database session, and set the member variable in the CwmsDatase class.

Reply:
Or just delete that comment